tests/Javascript/Pages/Admin/Roles/CreateRoleForm.spec.js

```diff
@@ -1,38 +1,37 @@
 import {afterEach, beforeEach, jest, test} from "@jest/globals";
 
 jest.mock('laravel-jetstream')
-import {createLocalVue, mount} from '@vue/test-utils'
+import {createLocalVue, shallowMount} from '@vue/test-utils';
 import {InertiaApp} from "@inertiajs/inertia-vue";
 import {InertiaForm} from "laravel-jetstream";
 import {InertiaFormMock} from "@test/__mocks__/laravel-jetstream";
 import CreateRoleForm from "@src/Pages/Admin/Roles/CreateRoleForm";
 
 let localVue;
-
+let wrapper;
 
 beforeEach(() => {
-    InertiaFormMock.error.mockClear()
-    InertiaFormMock.post.mockClear()
-
     localVue = createLocalVue()
     localVue.use(InertiaApp)
     localVue.use(InertiaForm)
 
+    wrapper = shallowMount(CreateRoleForm, {localVue})
```

Comment:
wrapper should be initialized as part of each test, and not in the `beforeEach` as the options passed to shallowMount may change depending on the test. 

Ex: A test may require `data` to be mocked, which is done via the `data:` field of the `options` parameter passed to `shallowMount`

Reply:
its possible to modify the component's data and props via helper methods after it already gets initialized. Hence why made this change.
<img width="888" alt="Screen Shot 2020-10-17 at 6 29 52 PM" src="https://user-images.githubusercontent.com/6550234/96354637-490ca080-10c8-11eb-9d06-9dd0765bfcf2.png">

Reply:
TIL!